skyportal/handlers/api/public_pages/public_source_page.py

```diff
@@ -22,19 +22,21 @@ def calculate_hash(data):
 
 
 def process_thumbnails(thumbnails, ra, dec):
+    # Sort thumbnails by type, and remove 'DR8' thumbnail if 'LS' (that corresponds to DR9) thumbnail are present
+    has_ls = any("ls" in thumbnail["type"] for thumbnail in thumbnails)
     thumbnails = sorted(
-        thumbnails,
+        [thumb for thumb in thumbnails if not has_ls or "dr8" not in thumb["type"]],
```

Comment:
though this is completely correct, would you mind changing it to: `if not (thumb["type"] == "dr8" and has_ls). Just much more clear and readable that way (and not at the cost of performance clearly).

Thanks!

Reply:
@antoine-le-calloch in case you missed this